app/models/concerns/profile_policy_association.rb

```diff
@@ -11,6 +11,7 @@ def policy
       return self unless external
 
       Profile.includes(:benchmark)
+             .older_than(created_at)
```

Comment:
Profile A is a policy of profile B only if profile A is older than profile B. This is Mohit's clever workaround.